extensions/amp-analytics/0.1/events.js

```diff
@@ -171,6 +187,17 @@ function isVideoTriggerType(triggerType) {
   return triggerType.startsWith('video');
 }
 
+/**
+ * @param {string} triggerType
+ * @return {boolean}
+ */
+function isCustomBrowserTriggerType(triggerType) {
+  const values = Object.keys(BrowserEventType).map((key) => {
+    return BrowserEventType[key];
+  });
```

Comment:
Don't do this. You're creating `values` every time this function is called. Instead create `values` outside of this function and assign to a var for faster lookups later.

Reply:
`isEnumValue` would be better, see [comment with suggestion.](https://github.com/ampproject/amphtml/pull/35193/files#r677585774)

Reply:
Sounds good. Thanks